assets/src/stories-editor/components/inserter/edit.css

```diff
@@ -0,0 +1,11 @@
+.amp__block-editor-inserter__menu.block-editor-inserter__menu {
```

Comment:
Upstream seems to use flexbox for this and the results. Does it make much of a difference?

https://github.com/WordPress/gutenberg/blob/47dfd82ce25e18458d6349cf9c40d94f8fdf2c9f/packages/block-editor/src/components/inserter/style.scss#L38-L41
https://github.com/WordPress/gutenberg/blob/47dfd82ce25e18458d6349cf9c40d94f8fdf2c9f/packages/block-editor/src/components/inserter/style.scss#L91-L110

Reply:
Hmm, will try an alternative with flexbox.